woocommerce/woocommerce-gutenberg-products-block
# Checkbox Control: Fix squished checkboxes when label text wraps (#5192)

On the front-end, checkboxes with labels would look squished when the labels wrapped on a second line. That was due to the flex container taking too much space for the label as the checkbox minimum dimensions was set in `px` rather than `em`, making it work differently depending on the relative font size.


Fixes # 5192

### Screenshots

*Before:*

![before](https://user-images.githubusercontent.com/1847066/142509470-72e3f025-42b4-4346-abf1-12de890c6316.png)

*After:*

![Screen Shot 2021-11-21 at 17 59 38](https://user-images.githubusercontent.com/1847066/142771623-041c93ad-ed46-4c81-8dbc-a8ffe2c4164f.png)

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Add a checkout block
2. Select Terms and Conditions block
3. Enable Require checkbox attribute
4. Preview your page
5. Make sure one label of the checkbox wraps to two lines on your screen.
6. Confirm square aspect ratio

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
